### PR TITLE
Allow models without select flag

### PR DESCRIPTION
**Pull request by turbo1912**

### CODE REVIEW

File: src/fal/cli/dbt_runner.py [hunk 1]
```diff
@@ -73,15 +72,26 @@ def dbt_run(args, models_list: List[str], target_path: str, run_index: int):
...
+    # make sure all are strings before joining for printing
+    command_list = list(map(str, command_list))
+
     # Execute the dbt CLI command in a subprocess.
-    full_command = " ".join(
-        command_list
-    )  # make sure all are strings before joining for printing
+    full_command = " ".join(command_list)
+
+    # make sure all are strings before joining for printing
```
Comment (chamini2) [inline src/fal/cli/dbt_runner.py:94]: remove this comment, it's copied from above and not relevant here.

File: src/fal/cli/dbt_runner.py [hunk 2]
```diff
@@ -73,15 +72,26 @@ def dbt_run(args, models_list: List[str], target_path: str, run_index: int):
     if args.selector:
```
Comment (chamini2) [inline src/fal/cli/dbt_runner.py:72]: Shouldn't we remove this too? I think a selector is processed in the fal flow level and then we pass the correct `--select` flags to dbt.

Comment (turbo1912) [reply src/fal/cli/dbt_runner.py:72]: yea i am not sure how we can support selectors right now seems like the currently code probably doesn't handle it perfectly either.. for example we dont handle tags.. ok creating a ticket for it

Comment (chamini2) [reply src/fal/cli/dbt_runner.py]: I am not sure it doesn't work, I haven't looked too much in detail inside fal flow selection flags process. I think it _should_ work? Do you want to add an integration test to test it?

Comment (turbo1912) [reply src/fal/cli/dbt_runner.py]: Adding tests in a different PR..

**Pull request merged**